ksqldb-execution/src/main/java/io/confluent/ksql/execution/util/CoercionUtil.java

```diff
@@ -101,11 +103,19 @@ private CoercionUtil() {
    * @param typeManager the type manager used to resolve expression types.
    * @return the coercion result.
    */
-  public static Result coerceUserList(
+  static Result coerceUserList(
```

Comment:
Did you keep this for testing purposes? IIUC all the places in code that this was used have been replaced by the other method. If it's there for testing, should we note that or leave it public?

Reply:
Let me rework the javadocs, this is just for testing purposes yes